pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -3034,7 +3035,7 @@ spec:
   pipelineRef:
     name: test-pipeline-with-finally
   timeouts:
-    tasks: 5m
+    tasks: 25m
```

Comment:
Why is there an increase in timeouts in this test? don't have a good understanding of the rest of the test, but shouldn't the time of the test not be increased ?

Reply:
yes, It should not increase. I think to fix the test had increased the time.
I check and fix the test.

Reply:
Hi @waveywaves, After the fix, test was failing with Pipelinerun timeout error as `pr.spec.timeouts.tasks` should be greater that `pr.spec.timeouts.pipeline` so I increased the `pr.spec.timeouts.tasks` value